app/server-utils/stampy.ts

```diff
@@ -258,6 +258,11 @@ const convertToQuestion = ({name, values, updatedAt} = {} as AnswersRow): Questi
 })
 
 export const loadQuestionDetail = withCache('questionDetail', async (question: string) => {
+  // Make sure all tags are loaded. This shouldn't be needed often, as it's double cached
+  if (Object.keys(allTags).length === 0) {
+    const {data} = await loadTags('NEVER_RELOAD')
+    allTags = Object.fromEntries(data.map((r) => [r.name, r])) as Record<string, Tag>
```

Comment:
I'm not sure whether this is the actual cause. Though I can't see anything else that might be an issue. It's a pity that Coda doesn't have proper joins, which would solve all of these issues...

My suspicion is that cloudflare changes workers, which means that the first one works fine, as it set `allTags` to contain the tags, but then:
* it calls `loadTags()` in index.tsx
* the cached value is returned (so it didn't actually call the function)
* `allTags == {}` for that worker

This will protect from that, at the potential cost of more calls

Reply:
I'm slightly worried about long term maintainability if we get into the habit of optimistic fixing / making workarounds without understanding root cause 🤔 

but currently on vacation and I don't see anything obviously bad about this PR itself, so YOLO approving 🤷

Reply:
agreed :/